thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js

```diff
@@ -16,7 +16,9 @@ export function formatEvalMetric(metric, isPercentage = false) {
   const multiplier = isPercentage ? 100 : 1;
   const convertedNum = metric * multiplier;
   const formattedNum = isWhole ? convertedNum : convertedNum.toFixed(1);
-  return isFinite(metric) ? formattedNum : shown;
+  let displayNum = isFinite(metric) ? formattedNum : shown;
+  // Prevent meaninglessly large numbers
+  return (Number(displayNum) > 10000) ? 'N/A' : displayNum;
```

Comment:
Arbitrary eliminator of excessively long performance numbers, which would be errors.